dashboard/app/models/sections/omni_auth_section.rb

```diff
@@ -61,7 +61,7 @@ def self.from_omniauth(code:, type:, owner_id:, students:, section_name: I18n.t(
     oauth_section.update(hidden: false) if oauth_section.hidden
 
     oauth_students = students.map do |student|
-      User.from_omniauth(student, {'user_type' => User::TYPE_STUDENT})
+      User.from_omniauth(student, {'user_type' => User::TYPE_STUDENT}, roster_sync: true)
```

Comment:
Since `roster_sync` is a user attribute, wouldn't it make more sense to pass it as a user parameter?
```suggestion
      User.from_omniauth(student, {'user_type' => User::TYPE_STUDENT, 'roster_sync' => true})
```

Reply:
Oh, that's a good point - also makes for one less parameter

Reply:
Good point, I think I initially thought those params were also used to search for an existing user, but that isn't the case. I'll go ahead and make that change, since it simplifies the function signature too.